pages/release_notes.rst

```diff
@@ -5,6 +5,54 @@ Release notes
 
 Whats has been changed, and how to update between versions.
 
+*******
+v0.13.0
+*******
+
```

Comment:
Perhaps add a link to the github project.

https://github.com/orgs/nuts-foundation/projects/8

Also, we can add the little project summary like:

> Focus of this relase was on robustness; automatic resumption of listeners/handlers, no more lost events or duplicate events. Developer Happiness by providing scripts to setup the nuts-local-network. Adding a Demo-EHR so nuts becomes clickable. Improving the registry by making it event based and adding signatures. Provide a convenient API authorization method by providing an easy to use OAuth 2 flow witch accepts IRMA identity tokens.

Reply:
added